### ISSUE TITLE
Support for Solidity library?

### ISSUE THREAD

**Issue by montyly** [crytic/echidna#173]
Echidna seems to fail with libraries:
```
library Test{
    struct Storage{
        uint val;
    }

    function set(Storage storage st) public{
        st.val = 501;
    }

}

contract Contract{
    using Test for Test.Storage;
    Test.Storage st;

    function set() public{
        st.set();
    }

    function echidna_test() external view returns (bool) {
        return st.val <= 500;
    }
}
```
Tested with the latest version on `eth-security-toolbox`:
```
$ echidna-test test.sol 
test.sol:1:1: Warning: Source file does not specify required compiler version! Consider adding "pragma solidity ^0.5.2;"
library Test{
^ (Relevant source part starts here and spans across multiple lines).
"Multiple contracts found in file, only analyzing the first"
"Analyzing contract: test.sol:Contract"
Passed property "echidna_test".
```

**Comment by montyly** [comment]
It seems that executing a library makes the function always revert.

For example, in the previous example if I add
```
    constructor() public{
    
    }
```
I have the same result, but if I add
```
    constructor() public{
        st.set();    
    }
```
This error occurs:
```
echidna-test: Call target doesn't exist
CallStack (from HasCallStack):
  error, called at src/EVM.hs:1136:9 in hevm-0.21
```

**Comment by ggrieco-tob** [comment]
I believe it is fixed in #224. Please test! :smile:

**Comment by ggrieco-tob** [comment]
This should be fixed. I will include in the wiki a more complete example to help people to test their contracts with libraries.
